djangocms_admin_style/sass/components/_content.scss

```diff
@@ -166,15 +166,42 @@ body {
         }
         a {
             font-size: $font-size-normal !important;
-            color: $gray-light;
+            color: $color-success;
             padding-left: 0 !important;
             background: none !important;
             &:before {
-                color: $color-primary;
+                color: $color-success;
             }
         }
     }
 }
+#taggit_taggeditem_items-group {
+    h2 {
+        font-weight: bold;
+        font-size: $font-size-large;
+    }
+    h3 {
+        color: $gray-darkest;
+        margin-top: 20px;
+    }
+    .module {
+        padding: 10px;
+        margin: 5px -10px 0;
+        border-top: solid 1px $gray-lighter !important;
+        border-bottom: solid 1px $gray-lighter !important;
+    }
+    .add-row {
+        margin-top: 10px;
+    }
+    .inline-deletelink {
+        color: $gray-darkest !important;
+        font: 0/0 a;
```

Comment:
font:0/0 a – a shorthand property that zeros out the font size and line-height. The a value acts as a very short font-family. It's used in bootstrap for hiding text as well